darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1336,6 +1401,12 @@ def load_model(path: str) -> "TorchForecastingModel":
             Path under which to save the model at its current state. The path should end with '.pth.tar'
         """
 
+        load_warning = (
+            "`TorchForecastingModel.load_model()` is deprecated and will be removed in a future darts "
+            "version. Please use `TorchForecastingModel.load()` instead."
+        )
+        raise_deprecation_warning(load_warning, logger)
+
         raise_if_not(
```

Comment:
Same comment here, don't you want to replace the code below by a call to `self.load()` ?